core/blockly.js

```diff
@@ -252,7 +252,7 @@ Blockly.onKeyDown_ = function(e) {
  * @private
  */
 Blockly.copy_ = function(block) {
-  var xmlBlock = Blockly.Xml.blockToDom(block);
+  var xmlBlock = (block.isComment) ? block.toXmlWithXY() : Blockly.Xml.blockToDom(block);
```

Comment:
Update the parameter annotation so that this can be a block | workspaceComment.  Rename the param to `toCopy` to indicate that it's required to be a block.  Similarly, `xmlBlock` isn't guaranteed to represent a block anymore.

What does `deleteNext` do if you have a comment instead of a block?

Reply:
Same thing for `Blockly.duplicate_`, actually.

Reply:
deleteNext does nothing, since there's no next block.